src/components/Person/Person.jsx

```diff
@@ -1 +1,23 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <>
+      <h2 className="Person__name">
+        {`My name is ${person.name}`}
+      </h2>
+
+      {person.age && (
+        <p className="Person__age">
+          {`I am ${person.age}`}
+        </p>
+      )}
+
+      <p className="Person__partner">
+        {person.isMarried === true ? (
+          `${person.partnerName} is my ${person.sex === 'm' ? 'wife' : 'husband'}`
+        ) : (
+          'I am not married'
+        )}
```

Comment:
1) move this conditions to a variable to increase readability
2) `person.isMarried === true ?` is the same as just `person.isMarried ?`